src/Swarm/Game/State.hs

```diff
@@ -609,6 +609,7 @@ scenarioToGameState scenario userSeed toRun g = do
       , _waitingRobots = M.empty
       , _gensym = initGensym
       , _randGen = mkStdGen seed
+      , _entityMap = g ^. entityMap <> scenario ^. scenarioEntities
```

Comment:
Btw. @byorgey does that not imply we would accumulate recipes from scenarios?

I am pretty sure that is what happens after going from drill challenge to any other scenario, just create rocks and you will see the iron vein in the recipes.

Reply:
Yes, we accumulate recipes from scenarios as well.  I agree this is not super problematic, it will never cause the game to crash or any scenarios to work incorrectly.  It is just sort of a way to cheat by first loading a scenario with some extra entities or recipes, just so you can have access to them in another scenario.